e2e/tests/init_test.go

```diff
@@ -193,6 +193,39 @@ func (s *suite) TheUserStakesAWithUpoktFromTheAccount(actorType string, amount i
 	require.NoError(s, err, "error creating config file in %q", path.Join(os.TempDir(), configPathPattern))
 
 	configContent := fmt.Sprintf(`stake_amount: %d upokt`, amount)
+	s.Log(configContent)
```

Comment:
OPTIONAL NIT: Did you intend to keep this in or was it just for testing?

If you want to keep it going, maybe have a subtitle of what it's for?